version.py

```diff
@@ -0,0 +1 @@
+vcf/__init__.py
```

Comment:
Can be deleted no?

Reply:
Had to reimplement how it `__init__.py` gets the version, but yeah, I don't think `version.py` is doing anything.

Reply:
In my revised version, it gets the VERSION from `pyproject.toml` (indirectly through `importlib.metadata`) whereas previously `__init__.py` was the source of authority on version.